src/plugins/LabelBot/strategies/configFlow.ts

```diff
@@ -0,0 +1,26 @@
+import { PRContext } from "../../../types";
+import { ParsedPath } from "../../../util/parse_path";
+
+export default function (context: PRContext, parsed: ParsedPath[]) {
+  const addedFlows = new Set(
+    parsed
+      .filter(
+        (fil) =>
+          fil.type == "component" &&
+          fil.status == "added" &&
+          fil.filename === "config_flow.py"
+      )
+      .map((fil) => fil.component)
+  );
+  // remove new integrations
+  for (const fil of parsed) {
+    if (
+      fil.type == "test" &&
```

Comment:
```suggestion
      fil.type == "component" &&
```
We want to delete for new integrations (new `__init__.py` for "component")